package/gluon-ebtables-split-igmpmld-domain/Makefile

```diff
@@ -0,0 +1,42 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-ebtables-split-igmpmld
```

Comment:
The directory is called *gluon-ebtables-split-igmpmld-domain*, but the package name is *gluon-ebtables-split-igmpmld*.